app/services/idv/flows/capture_doc_flow.rb

```diff
@@ -25,6 +25,16 @@ def initialize(controller, session, _name)
       def flow_path
         'hybrid'
       end
+
+      def extra_analytics_properties
+        bucket = AbTests::NATIVE_CAMERA.bucket(flow_session[:document_capture_session_uuid])
```

Comment:
Just want to double-check: Is this the right place to be looking for the UUID in the hybrid flow? In the controller I see it being referenced on `session` (not `flow_session`):

https://github.com/18F/identity-idp/blob/58ef8e2242425ee7d8cf50a8ec54087954eb268d/app/controllers/idv/capture_doc_controller.rb#L46

Reply:
Great catch - updated in 5a371e6aa

Reply:
I don't see the change in 5a371e6 ?

Reply:
This is why I shouldn't push changes late on a Sunday! 🤦 Amended the commit in eb2a390b5